File: ruby/arrays/functions_operating_on_arrays/operations_on_elements/append.rb
#!/usr/bin/ruby2.7

array = [2, 4, 6]

puts("Initial state:\n\n")
puts(array.inspect, "\n")

r = array.append(8)

puts("After append by array.append(8):\n\n")
puts(array.inspect, "\n")
puts("Returned value: #{r}\n\n")

r = array.append(10, 12, 14)

puts("After append by array.append(10, 12, 14):\n\n")
puts(array.inspect, "\n")
puts("Returned value: #{r}\n\n")

File: python/arrays/lists/functions_operating_on_lists/operations_on_elements/append.py
# Copyright (C) 2019-2026 Katarzyna Krasińska
# Languagium - https://github.com/katheroine/languagium
# Licensed under GPL-3.0 - see LICENSE.md
# SPDX-License-Identifier: GPL-3.0-or-later

from pprint import pprint

list = [2, 4, 6]

print("Initial state:\n")
pprint(list); print()

r = list.append(8)

print("After append by list.append(8):\n")
pprint(list); print()
print(f"Returned value: {r}\n\n")
